src/serac/numerics/mesh_utils.cpp

```diff
@@ -461,7 +461,11 @@ serac::mesh::InputOptions FromInlet<serac::mesh::InputOptions>::operator()(const
       overall_size = std::vector<double>(overall_size.size(), 1.);
     }
 
-    return {serac::mesh::GenerateInputOptions{elements, overall_size}, ser_ref, par_ref};
+    return {serac::mesh::BoxInputOptions{elements, overall_size}, ser_ref, par_ref};
+  } else if (mesh_type == "disk" || mesh_type == "ball") {
+    int approx_elements = base["approx_elements"];
+    int dim             = (mesh_type == "disk") ? 2 : 3;
```

Comment:
I generally don't like using ternary operators unless there's a good reason. I think an if statement is clearer. However, I don't feel extremely strongly about this.